handbook/project-management/scope.md

```diff
@@ -1,27 +1,66 @@
-# 🔍 Project Scope
+# 🔍 Project Scope <Badge type="tip" text="updated"/>
 
-This document outlines advice on how to manage projects requirements and scope.
+This document outlines advice on how to scope your project in a way that makes it easier to deliver on the (comparatively) tight schedules Launch Pad projects have.
 
-## Minimum Viable Product (MVP)
+## Launch Pad Limitations
 
-A project's scope should be **as minimal as possible** while maintaining core
-functionality. The scope should be defined in terms of must-have features
-("requirements") and bonus features ("stretch goals"). As much as possible, a
-project's set of requirements should fully describe the necessary steps to
-building the project.
+Note that on paper, it may seem like you have a lot of time since teams get a whole semester to complete their project. However, in between designing your project, learning the prerequisite technologies, adapting to feedback, midterms, breaks, absences, and more, teams don't actually have a lot of time to complete projects.
```

Comment:
Note that on paper, it may seem like you have a lot of time since teams get a whole semester to complete their project. However, in between designing your project, learning the prerequisite technologies, adapting to feedback, midterms, breaks, absences, and more, many previous teams actually struggled to complete the projects.